scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -1686,38 +1698,49 @@ class FormatOps(
       def penalize(penalty: Int) =
         if (penalty <= 0) Policy.NoPolicy
         else new PolicyOps.PenalizeAllNewlines(Policy.End.On(blast), penalty)
-      def getNlSplit(penalty: Int)(implicit line: sourcecode.Line): Split = {
+      def getNlSplit(penalty: Int)(implicit
+          line: sourcecode.Line,
+          fileName: sourcecode.FileName
+      ): Split = {
         val nlLine = line.copy(value = line.value + 1)
-        nlSplitFunc(1).andPolicy(penalize(penalty)).forThisLine(nlLine)
+        nlSplitFunc(1)
+          .andPolicy(penalize(penalty))
+          .forThisLine(nlLine, fileName)
```

Comment:
suggest adding the following method in `object FormatOps`:
```
  def nextLine(implicit fl: FileLine): FileLine = {
    val line = fl.line
    new FileLine(fl.file, line.copy(value = line.value + 1))
  }
```
and using it here and in another place.

Reply:
Added